osu.Game.Rulesets.Taiko/Difficulty/TaikoDifficultyCalculator.cs

```diff
@@ -90,19 +98,21 @@ protected override DifficultyAttributes CreateDifficultyAttributes(IBeatmap beat
             double monoStaminaRating = singleColourStamina.DifficultyValue() * stamina_skill_multiplier;
             double monoStaminaFactor = staminaRating == 0 ? 1 : Math.Pow(monoStaminaRating / staminaRating, 5);
 
+            double colourDifficultStrains = colour.CountTopWeightedStrains();
+            double rhythmDifficultStrains = rhythm.CountTopWeightedStrains();
+            double staminaDifficultStrains = stamina.CountTopWeightedStrains();
+
             double combinedRating = combinedDifficultyValue(rhythm, colour, stamina, isRelax);
             double starRating = rescale(combinedRating * 1.4);
 
-            // TODO: This is temporary measure as we don't detect abuse of multiple-input playstyles of converts within the current system.
+            // Converts are penalised outside the scope of difficulty calculation, as our assumptions surrounding standard play-styles becomes out-of-scope.
             if (beatmap.BeatmapInfo.Ruleset.OnlineID == 0)
             {
-                starRating *= 0.925;
-
-                // For maps with either relax or low colour variance and high stamina requirement, multiple inputs are more likely to be abused.
+                // For maps with relax, multiple inputs are more likely to be abused.
                 if (isRelax)
                     starRating *= 0.60;
-                else if (colourRating < 2 && staminaRating > 8)
-                    starRating *= 0.80;
```

Comment:
Are we confident this won't break the world again? 😅

Reply:
Also, what "bump" to convert penalties are you talking about? I don't see any convert-related changes besides this one.